package.json

```diff
@@ -37,7 +37,10 @@
   },
   "homepage": "https://github.com/donmccurdy/aframe-physics-system#readme",
   "dependencies": {
+    "ammo-debug-drawer": "github:infinitelee/ammo-debug-drawer",
+    "ammo.js": "github:kripken/ammo.js#ebe912c0426a47da9a11ce1ded3815da4a8d27c1",
```

Comment:
Should Ammo be a peer dependency instead? It seems more appropriate to me, since aframe-physics-system makes no attempt to load Ammo on behalf of the application, and since users of aframe-physics-system who continue to use Cannon will have no desire to take Ammo as a dependency.

Reply:
Or perhaps even not a dependency at all, and we document the recommended installation. Peer dependencies are technically optional, but trying to get the `require`s right with everybody's package managers sounds like a mess. 😕

A recommended, tagged package like `@aframevr/ammo.js` and `@aframevr/cannon.js` would be ideal IMO, since cannonjs is on a custom SHA1 too.

Reply:
I removed ammo.js as a dependency. I'll add to the documentation (when I finally get around to updating it) options for including ammo.js